gha/py-dependencies/action.yaml

```diff
@@ -55,48 +59,52 @@ runs:
         ./scripts/github/install-virtualenv.sh
         PIP_VERSION=$(grep '^PIP_VERSION' Makefile | awk '{print $3}')
         echo "::set-output name=pip-version::${PIP_VERSION}"
-        ${VIRTUALENV_DIR}/bin/pip install -U "pip==${PIP_VERSION}" setuptools
+        ${{env.VIRTUALENV_DIR}}/bin/pip install -U "pip==${PIP_VERSION}" setuptools
```

Comment:
The `${{ }}` vs `${ }` bits are more problematic here. If VIRTUALENV_DIR is not available here, then install-virtualenv.sh might not work correctly.

Reply:
Just so you see the comparison here too:
![Uploading single-vs-double.png…]()

Reply:
Just for reference:
![single-vs-double](https://user-images.githubusercontent.com/92544319/143135282-90805ee6-cf9f-40a0-a1bb-c189b68a1dd0.png)